packages/fx-core/src/component/driver/botAadApp/create.ts

```diff
@@ -117,8 +117,8 @@ export class CreateBotAadAppDriver implements StepDriver {
 
   private loadCurrentState(): CreateBotAadAppOutput {
     return {
-      BOT_ID: process.env.BOT_ID,
-      SECRET_BOT_PASSWORD: process.env.SECRET_BOT_PASSWORD,
+      MICROSOFT_APP_ID: process.env.MICROSOFT_APP_ID,
```

Comment:
I'm not sure whether we need to change the output here: users will see both `AAD_APP_CLIENT_ID` and `MICROSOFT_APP_ID` in .env file. Without certain context, this may be confusing to users.

Reply:
I'm considering to add comments here.  @MuyangAmigo may take a look.